### ISSUE TITLE
Issue during running

### ISSUE THREAD

**Issue by lasaths** [Jan0660/DocFxMarkdownGen#2]
Hi what could be causing this error? It starts to run and a few md scripts are made

```
[INFO] DocFxMarkdownGen v0.1.5 running...
[INFO] Read all YAML in 66ms.
[INFO] Generating and writing markdown...
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at Program.<<Main>$>g__SourceLink|0_10(Item item) in C:\Users\Jan\RiderProjects\DocFxMarkdownGen\Program.cs:line 132
   at Program.<<Main>$>g__Declaration|0_11(StringBuilder str, Item item) in C:\Users\Jan\RiderProjects\DocFxMarkdownGen\Program.cs:line 136
   at Program.<>c__DisplayClass0_0.<<<Main>$>b__12>d.MoveNext() in C:\Users\Jan\RiderProjects\DocFxMarkdownGen\Program.cs:line 161
--- End of stack trace from previous location ---
   at System.Threading.Tasks.Parallel.<>c__50`1.<<ForEachAsync>b__50_0>d.MoveNext()
--- End of stack trace from previous location ---
   at Program.<Main>$(String[] args) in C:\Users\Jan\RiderProjects\DocFxMarkdownGen\Program.cs:line 145
   at Program.<Main>(String[] args)
zsh: abort      dfmg
```

**Comment by Jan0660** [comment]
Are you running this on something that does not have a Git repo with a remote? I couldn't reproduce the issue with a project without a Git repository, which should be the way.
If you are able to, please provide me with a link to the repo you were trying to create docs for.

Just released v0.1.6, it should skip trying to get a source link instead of crashing.

**Comment by lasaths** [comment]
Amazing! It works!

I think that might of been the issue. I had made a blank branch and didnt publish it to remote.
